### ISSUE TITLE
Injectable is not a function

### ISSUE THREAD

**Issue by doncorsean** [angular/angular#14562]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
When we bundle our app or load dynamically no decorators are found on objects.
i.e. 
```
Uncaught TypeError: core_1.Injectable is not a function
```

**Expected behavior**
App would run normally with no errors

**Minimal reproduction of the problem with instructions**
https://github.com/doncorsean/angular-demo
npm install
gulp buildDynamic
npm install http-server -g
http-server

**Please tell us about your environment:**
Red Hat, IDEA, npm, Play over Nginx

* **Angular version:** 2.4.7
    "@angular/common": "^2.4.7",
    "@angular/compiler": "^2.4.7",
    "@angular/core": "^2.4.7",
    "@angular/forms": "^2.4.7",
    "@angular/http": "^2.4.7",
    "@angular/platform-browser": "^2.4.7",
    "@angular/platform-browser-dynamic": "^2.4.7",
    "@angular/router": "^3.4.7",

* **Browser: all
 
Language: TypeScript

**Comment by doncorsean** [comment]
issue was a combination of systemjs version and systemjs builder version

**Comment by snarum** [comment]
I'm having the same problem. do you remember how you discovered what was wrong, and maybe even how you solved tid? I have installer jspm 0.17.0.beta-40, and i guess system.js comes with that?

**Comment by doncorsean** [comment]
Was an issue with incompatible versions of systemjs/systemjsbuilder iirc

Note for SystemJS 0.19 support use SystemJS Builder 0.15

Corey Ricketts
Mobile: 305-342-3056
corey@oceandrivegraphics.com<mailto:corey@oceandrivegraphics.com>
http://www.oceandrivegraphics.com

On Mar 10, 2017, at 7:26 AM, snarum <notifications@github.com<mailto:notifications@github.com>> wrote:


I'm having the same problem. do you remember how you discovered what was wrong, and maybe even how you solved tid? I have installer jspm 0.17.0.beta-40, and i guess system.js comes with that?

-

**Comment by snarum** [comment]
I finally fixed this, and it was caused by some inconsistancy between SystemJS 0.20.x and Angular2. After upgrading jspm to 0.17.0-beta40 systemJS was upgraded to 0.20.9 and to handle typescript and angular2 to correctly I had to add something like this to my jspm.config.js file:

    meta:{
      "@angular/core",{esModule:true},
     ...

some usefull information here:

https://github.com/frankwallis/plugin-typescript/issues/185
